packages/runtime/src/components/builtin/Form/components/Field/index.tsx

```diff
@@ -85,6 +85,11 @@ type Props = {
   tableFormField: TableFormField
 }
 
+// Escape characters: [ ] '
+function escapeCharacters(string: string) {
+  return string.replace(/[[\]']/g, '\\$&')
```

Comment:
Doesn't the `[` need to be escaped as well? The syntax highlighting seems to be giving that off as well.

Reply:
Shouldn't be. I got this function from [here](https://stackoverflow.com/questions/3561493/is-there-a-regexp-escape-function-in-javascript/3561711#3561711), and it looks like it's working on [regExr](https://regexr.com/)